content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -80,6 +80,14 @@ Used on: All Objects
 
 This annotation is used for describing specific behaviour of given object.
 
+## kubernetes.io/enforce-mountable-secrets {#enforce-mountable-secrets}
+
+Example: `kubernetes.io/enforce-mountable-secrets: "true"`
+
+Used on: ServiceAccount
+
+The value for this annotation must be **true** to take effect. This annotation indicates that a ServiceAccount should enforce mountable secrets. This field should not be used to find auto-generated service account token secrets for use outside of pods.
```

Comment:
```suggestion
The value for this annotation must be **true** to take effect.
This annotation indicates that pods running as this service account may only reference
Secret API objects specified in the service account's `secrets` field.
```

Reply:
Thanks, @liggitt for taking look at this.
this seems good, I will change it according to your suggestion.

Reply:
Hi @liggitt
I have addressed the comment, PTAL!

Thanks!